hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: olm-artifacts-template
+
+parameters:
+- name: REGISTRY_IMG
+  required: true
+- name: CHANNEL
+  value: staging
+- name: IMAGE_TAG
+  value: latest
+
+objects:
+- apiVersion: operators.coreos.com/v1alpha1
+  kind: CatalogSource
+  metadata:
+    name: managed-velero-operator-catalog
+  spec:
+    sourceType: grpc
+    image: ${REGISTRY_IMG}:${CHANNEL}-${IMAGE_TAG}
+    displayName: managed-velero-operator Registry
+    publisher: SRE 
+
+- apiVersion: operators.coreos.com/v1alpha2
+  kind: OperatorGroup
+  metadata:
+    name: managed-velero-operator-og
+  spec:
+    targetNamespaces:
+    - managed-velero-operator
+
+- apiVersion: operators.coreos.com/v1alpha1
+  kind: Subscription
+  metadata:
+    name: managed-velero-operator
+  spec:
+    channel: ${CHANNEL}
+    name: managed-velero-operator
+    source: managed-velero-operator-catalog
+    sourceNamespace: managed-velero-operator
```

Comment:
@dak1n1 This needs to be a selector sync set containing the OLM object :)  like https://github.com/openshift/configure-alertmanager-operator/blob/master/build/templates/olm-artifacts-template.yaml.tmpl

Reply:
You'll also need to create the namespace, role, robebinding and service account, also specify the namespaces in the OLM spec for the CatalogSource, OperatorGroup and Subscription see above dedicated-admin repo